pharus/component_interface.py

```diff
@@ -254,8 +330,11 @@ def dj_query_route(self):
 
 
 type_map = {
+    "query": QueryComponent,
     "plot:plotly:stored_json": PlotPlotlyStoredjsonComponent,
     "table": TableComponent,
     "metadata": MetadataComponent,
     "file:image:attach": FileImageAttachComponent,
+    "slider": SliderComponent,
+    "dropdown-query": SliderComponent,
```

Comment:
Unclear why it was necessary to add the `query`, `slider`, and the `dropdown-query` component here.